### ISSUE TITLE
Legal review of terms and conditions for Google Forms

### ISSUE THREAD

**Issue by scayzer** [bcgov/cloud-pathfinder#1199]
**Describe the issue**
In the course of investigating government-wide demand for AirTable vs alternate eForms solutions (ticket #1158), we found that SoAR and PIA have been done for Google Forms (by GRS) and it has several key advantages over AirTable for our onboarding Forms (strong support for validation of fields entered and ability for submitters to amend their submissions after hitting the submit button). The onboarding forms are now available in Google Forms and we just need to complete the PIA, SoAR  (ticket #1198) and legal review of terms and conditions.

**Additional context**
- Question for Audra to ask Douglas: would it make sense to do legal review of terms for Google Forms in parallel with negotiation of additional terms for GCFA? Or after?

**Definition of done**
- Legal review of Google Forms terms and conditions completed and any add/change to terms negotiated with Google.

**Comment by ActionAnalytics** [comment]
There is some CRM consolidation work in flight run by Daniel P & Audra & friends.  This is aiming to become real in a few months.  Since we need something to operate with for the time being, we will proceed with gForms but not go further with Legal until we have evaluated the new CRM to see if it works for us.

**Comment by wrnu** [comment]
No longer relevant
